db/migrate/20220117104521_change_currency_types.rb

```diff
@@ -0,0 +1,22 @@
+class ChangeCurrencyTypes < ActiveRecord::Migration[7.0]
+  def up
+    change_column :case_logs, :earnings, :decimal, precision: 10, scale: 2
```

Comment:
Just wondering, is there a case for using `bigint` instead? A scale of 2 is fine as long as we don't do any operations with rounding.

Reply:
How would using bigint work?

Reply:
Don't we collect all financial amounts to the nearest £ anyway? Why do we need to store a decimal?